patternfly/patternfly-elements
# feat: Add start to dev command; open to specific demo page

## Tooling update

- Add the start command to the `npm run dev` script; update documentation
- Add logic to spandx to fire a specific demo page if watching 1 component

## Testing instructions

1. `npm run dev` -> should compile the entire project, set a watch on all components, and fire the build server at: `http://localhost:8000/examples/`

2. `npm run dev pfe-accordion` -> should compile only pfe-accordion (and it's dependencies), set a watch on only pfe-accordion (and it's dependencies), and fire the build server at: `http://localhost:8000/elements/pfe-accordion/demo`

3. `npm run dev pfe-accordion pfe-icon` -> should compile only pfe-accordion and pfe-icon (and their dependencies), set a watch on only pfe-accordion and pfe-icon (and their dependencies), and fire the build server at: `http://localhost:8000/examples/`

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.


### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
